src/common/image.cpp

```diff
@@ -463,6 +463,19 @@ wxImage::Scale( int width, int height, wxImageResizeQuality quality ) const
     // Resample the image using the method as specified.
     switch ( quality )
     {
+        case wxIMAGE_QUALITY_NORMAL:
+            // When downscaling, we prefer to use bilinear resampling as it
+            // results in better quality at reasonable speed.
+            if ( width < old_width && height < old_height )
```

Comment:
```suggestion
            if ( width < old_width || height < old_height )
```

Reply:
Sorry, why? My understanding is that we want to avoid bilinear when increasing size, even in one of the directions.

One thing we could probably change here is what to do in case of equality, i.e.

```suggestion
            if ( width <= old_width && height <= old_height )
```

Reply:
>that we want to avoid bilinear when increasing size, even in one of the directions.

Seems safer to prefer bilinear resampling to me. Doesn't really matter though.

Reply:
If this improves things for KiCad icons, let's merge it already and we can always improve it later IMO.

Reply:
Sure, thanks!